interfaces/repo.go

```diff
@@ -656,3 +656,35 @@ func (r *Repository) RemoveSnap(snapInfo *snap.Info) error {
 
 	return nil
 }
+
+// DisconnectSnap disconnects all the connections to and from a given snap.
+//
+// The return value is a list of snap.Info's that were affected.
```

Comment:
You of course totally right, sorry. The problem isn't the local snap being removed, but the other ones (stating to myself). Thanks.